src/validator-guides/how-to-setup-a-validator-node.md

```diff
@@ -0,0 +1,79 @@
+
+From the repository's root directory execute following commands in order:
+
+```
+docker run -it --rm --name ggx-local-node -p 9944:9944 -p 9933:9933 -p 30333:30333 -v $(pwd)/custom-spec-files:/tmp golden-gate-node:sydney /usr/src/app/target/release/golden-gate-node --base-path=/data-sydney --chain /tmp/sydney.json --bootnodes /ip4/3.69.173.157/tcp/30333/p2p/12D3KooWSriyuFSmvuc188UWqV6Un7YYCTcGcoSJcoyhtTZEWi1n --telemetry-url "wss://dev.telemetry.sydney.ggxchain.io/submit 0"
+```
+
+You can use the following optional flags:
+
+```
+| Flags | Description |
+|---|---|
+| `--validator` | Starts the node with the authority role and enables it to actively <br>participate in any consensus task that it can (for example, depending on<br> availability of local keys). |
+| `--rpc-external` | Listens to all RPC interfaces. By default, the node only listens to <br>local RPC calls. If you set this command-line option, keep in mind that <br>that not all RPC methods are safe to be exposed publicly. Use an RPC <br>proxy server to filter out dangerous methods. For more information about<br> RPC methods that shouldn't be publicly exposed, see <br>Remote procedure calls. <br>Use `--unsafe-rpc-external` to suppress the warning if you understand the risks. |
+| `--ws-externa` | Listens to all Websocket interfaces. By default, the node only listens <br>locally. Keep in mind that not all RPC methods are safe to be exposed <br>publicly. You can use an RPC proxy server to filter out dangerous <br>methods. You can use `--unsafe-ws-external` to suppress the warning if you understand the risks. |
+| `--unsafe-rpc-external` | Listens to all RPC interfaces. This option is the same as <br>`--rpc-external`. |
+| `--unsafe-ws-external` | Listens to all Websocket interfaces. This option is the same as <br>`--ws-external` but doesn't warn you about it. |
+| `--base-path <path>` | Specifies a custom base path. |
+| `--bootnodes <node-identifier>` | Specifies a list of boot nodes identifiers for peer-to-peer communication. |
+| `--chain <chain-specification>` | Specifies the chain specification to use. You can set this option using a predefined chain specification name,<br>such as `dev`, `local`, or `staging`or you can specify the path to a file that contains the chain <br>specification, for example, the chain specification generated by using the build-spec subcommand. |
+| `--name <name>` | Specifies the human-readable name for this node. The node name is reported to the telemetry server, if enabled. |
+| `--password <password>` | Specifies the password to use for the keystore. |
+| `--telemetry-url <url verbosity>` | Specifies the URL of the telemetry server to connect to. You can pass <br>this flag multiple times to specify multiple telemetry endpoints. <br>Verbosity levels range from 0-9, with 0 denoting the least verbose. Use <br>the following format to specify the URL followed the verbosity option is `--telemetry-url 'wss://foo/bar 0'`. |
+```
```

Comment:
```suggestion
| Flags | Description |
|---|---|
| `--validator` | Starts the node with the authority role and enables it to actively <br>participate in any consensus task that it can (for example, depending on<br> availability of local keys). |
| `--rpc-external` | Listens to all RPC interfaces. By default, the node only listens to <br>local RPC calls. If you set this command-line option, keep in mind that <br>that not all RPC methods are safe to be exposed publicly. Use an RPC <br>proxy server to filter out dangerous methods. For more information about<br> RPC methods that shouldn't be publicly exposed, see <br>Remote procedure calls. <br>Use `--unsafe-rpc-external` to suppress the warning if you understand the risks. |
| `--ws-externa` | Listens to all Websocket interfaces. By default, the node only listens <br>locally. Keep in mind that not all RPC methods are safe to be exposed <br>publicly. You can use an RPC proxy server to filter out dangerous <br>methods. You can use `--unsafe-ws-external` to suppress the warning if you understand the risks. |
| `--unsafe-rpc-external` | Listens to all RPC interfaces. This option is the same as <br>`--rpc-external`. |
| `--unsafe-ws-external` | Listens to all Websocket interfaces. This option is the same as <br>`--ws-external` but doesn't warn you about it. |
| `--base-path <path>` | Specifies a custom base path. |
| `--bootnodes <node-identifier>` | Specifies a list of boot nodes identifiers for peer-to-peer communication. |
| `--chain <chain-specification>` | Specifies the chain specification to use. You can set this option using a predefined chain specification name,<br>such as `dev`, `local`, or `staging`or you can specify the path to a file that contains the chain <br>specification, for example, the chain specification generated by using the build-spec subcommand. |
| `--name <name>` | Specifies the human-readable name for this node. The node name is reported to the telemetry server, if enabled. |
| `--password <password>` | Specifies the password to use for the keystore. |
| `--telemetry-url <url verbosity>` | Specifies the URL of the telemetry server to connect to. You can pass <br>this flag multiple times to specify multiple telemetry endpoints. <br>Verbosity levels range from 0-9, with 0 denoting the least verbose. Use <br>the following format to specify the URL followed the verbosity option is `--telemetry-url 'wss://foo/bar 0'`. |
```

Reply:
Formatted the table